yaml/training_prod/ingress_ch.yaml

```diff
@@ -3,15 +3,15 @@ kind: Ingress
 metadata:
   annotations:
     field.cattle.io/publicEndpoints: '[{"addresses":["2a06:c00::37e9"],"port":443,"protocol":"HTTPS","serviceName":"pitc-ansible-training-prod:latest-ansible-puzzle","ingressName":"pitc-ansible-training-prod:latest-ansible-puzzle","hostname":"ansible.puzzle.ch","path":"/","allNodes":true}]'
-    kubernetes.io/ingress.class: nginx-public
     meta.helm.sh/release-name: latest
     meta.helm.sh/release-namespace: pitc-ansible-training-prod
   labels:
     app.kubernetes.io/instance: latest
     app.kubernetes.io/managed-by: Helm
     app.kubernetes.io/name: ansible-puzzle
     app.kubernetes.io/version: 1.0.0
-    helm.sh/chart: acend-training-chart-0.1.2
+    helm.sh/chart: acend-training-chart-0.1.4
```

Comment:
You shouldn't add helm labels if the resource is not managed by helm. Additionally, these labels would be set by helm itself.

Also, putting this resource definition here doesn't make a lot of sense. It should be created by the GitHub Actions jobs.

Reply:
I removed this file. Only the ingress_de.yaml is necessary because this one won't be handled by github actions.